src/Databases/PostgreSQL/DatabasePostgreSQL.cpp

```diff
@@ -182,10 +203,12 @@ void DatabasePostgreSQL::attachTable(const String & table_name, const StoragePtr
     std::lock_guard<std::mutex> lock{mutex};
 
     if (!checkPostgresTable(table_name))
-        throw Exception(fmt::format("Cannot attach table {}.{} because it does not exist", database_name, table_name), ErrorCodes::UNKNOWN_TABLE);
+        throw Exception(fmt::format("Cannot attach PostgreSQL table {} because it does not exist in PostgreSQL",
```

Comment:
btw, we have a special overload ctor
```
        throw Exception(ErrorCodes::UNKNOWN_TABLE, "Cannot attach PostgreSQL table {} because it does not exist in PostgreSQL", getTableNameForLogs(table_name), database_name));
```